app/views/shell/_top.html.erb

```diff
@@ -3,7 +3,7 @@
   <head>
     <meta charset="utf-8">
     <% title = yield(:title) %>
-    <title><%= title || "#{ApplicationConfig['COMMUNITY_NAME']} Community" %></title>
+    <title><%= CGI.unescapeHTML(title) || "#{ApplicationConfig['COMMUNITY_NAME']} Community" %></title>
```

Comment:
I'm not sure if the `||` outcome is truly never called or we just don't have appropriate test coverage, but it seems like this would result in an error if `title` is `nil`. So either `title` is never `nil` and we don't need the or statement, or we should `CGI. unescapeHTML` the whole thing.